semantic-segmentation-deeplabv3/script/dataset.py

```diff
@@ -0,0 +1,196 @@
+import os
+import random
+import numpy as np
+from PIL import Image, ImageOps, ImageFilter
+
+import torch
+
+
+class SegmentationDataset(object):
+    """Segmentation Base Dataset"""
+
+    def __init__(self, root, split, mode, transform, base_size=520, crop_size=480):
+        super(SegmentationDataset, self).__init__()
+        self.root = root
+        self.transform = transform
+        self.split = split
+        self.mode = mode if mode is not None else split
+        self.base_size = base_size
+        self.crop_size = crop_size
+
+    def _val_sync_transform(self, img, mask):
```

Comment:
How about moving funcs like this transform to a separate py?

Reply:
Transform is part of dataset. I don't think it's necessary to separate it.